src/settings/Settings.ts

```diff
@@ -120,6 +120,13 @@ export interface ISetting {
 }
 
 export const SETTINGS: {[setting: string]: ISetting} = {
+    "feature_spaces": {
+        isFeature: true,
+        displayName: _td("Spaces prototype. Incompatible with Communities, Communities v2 and Custom Tags"),
```

Comment:
If it's going to have full stops, let's use them for all phrases.

```suggestion
        displayName: _td("Spaces prototype. Incompatible with Communities, Communities v2 and Custom Tags."),
```

Reply:
Should this say it requires a compatible homeserver, like the communities v2 flag?

Reply:
I guess so, partially. "Some parts of this feature require a compatible homeserver" - I'm all for making the description longer and making this Labs flag stand out even more :P